drivers/filesystems/cdfs_new/cdstruc.h

```diff
@@ -170,6 +170,12 @@ typedef PVOID PBCB;     //**** Bcb's are now part of the cache module
 
 #define BYTE_COUNT_EMBEDDED_NAME        (32)
 
+#ifdef __REACTOS__
+#define __volatile
```

Comment:
Define it to volatile. To avoid unexpected optimizations.

Reply:
True, I meant to do something about those defines and then I forgot about them. Will do.